src/SigFox.cpp

```diff
@@ -122,6 +122,12 @@ int SIGFOXClass::begin(SPIClass& spi, int reset, int poweron, int interrupt, int
 int SIGFOXClass::send(unsigned char mess[], int len, bool rx)
 {
   if (len == 0) return -1;
+
+  if (rx == false && len == 1 && mess[0] < 2) {
+    //we can use send_bit command
```

Comment:
As far as I understand, on the backend, the received value becomes an `uint8_t`, it is only sent as a single bit at the physical layer.